packit_service/worker/handlers/github_handlers.py

```diff
@@ -459,19 +463,33 @@ class GithubTestingFarmHandler(JobHandler):
     trigger is finished copr build.
     """
 
-    triggers = [TheJobTriggerType.pull_request]
+    triggers = [
+        TheJobTriggerType.pull_request,
+        TheJobTriggerType.release,
+        TheJobTriggerType.commit,
+    ]
 
     def __init__(
         self,
         package_config: PackageConfig,
         job_config: JobConfig,
         data: EventData,
         chroot: str,
+        build_id: int,
     ):
         super().__init__(
             package_config=package_config, job_config=job_config, data=data,
         )
         self.chroot = chroot
+        self.build_id = build_id
+        self._db_trigger: Optional[AbstractTriggerDbType] = None
+
+    @property
+    def db_trigger(self) -> Optional[AbstractTriggerDbType]:
+        if not self._db_trigger:
+            build = CoprBuildModel.get_by_id(self.build_id)
```

Comment:
Just to be sure. What about returning `None` when no build is found?

Reply:
I was thinking about this, but in CoprBuildEndHandler if the build is not found, the testing farm handler task is not called.

Reply:
ok